CHANGES

```diff
@@ -1,3 +1,11 @@
+Changes from 8.8.2 to 8.9+beta1
```

Comment:
@gares what is Coq 8.8.2 ?

Reply:
I suppose this is expecting two minor releases like that was done for Coq 8.7. We used to write "Changes beyond 8.8" instead which wasn't putting any expectation.